tests/DocumentConverter.Tests/DocumentConverter.Tests.csproj

```diff
@@ -49,7 +49,7 @@
     <PackageReference Include="NUnit.Analyzers" Version="3.3.0" />
     <PackageReference Include="coverlet.collector" Version="3.1.2" />
     <PackageReference Include="Shouldly" Version="4.1.0" />
-    <PackageReference Include="Svg" Version="2.4.3.14" />
+    <PackageReference Include="Svg" Version="2.4.3.15" />
```

Comment:
pleasae check for the text-rendering bug first (no colour is appled to text fill in 2.4.3.15)